Ventilators/R4/fsh-tank/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
What are these files in fsh-tank? They are not retrievable through CRD, so what are they used for?

Reply:
Those are files for FHIR Shorthand. I used those to test auto generate Questionnaire by FSH. CRD does not need to read those file. I think I should remove those from this check in.

Reply:
Additional FSH files are removed